include/SQuIDS/SUNalg.h

```diff
@@ -530,14 +530,45 @@ class SU_vector{
   ///               greater than zero allows for a gradual transition to cut-off, rather
   ///               than applying a hard step function.
   void LowPassFilter(double* buffer, double cutoff, double scale) const{
+    if (fabs(scale) > fabs(cutoff))
```

Comment:
`std::abs` would be preferrable to `fabs`.

Reply:
ok I changed it